<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="BasicUIAutoTestSuiteCaseBasicFl"
           tests="9"
           failures="7"
           errors="0"
           skipped="0"
           time="58.569">
  <properties />
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.006" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.005" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.007" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="46.149" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.005" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.006" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="0.007" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [false] but found [true]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl.BasicUIAutoTestSuiteCaseBasicFlTest(BasicUIAutoTestSuiteCaseBasicFl.java:125)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="5.711" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl" />
                <testcase name="BasicUIAutoTestSuiteCaseBasicFlTest" time="6.673" classname="com.stta.BasicUIAutoTestSuite.BasicUIAutoTestSuiteCaseBasicFl" />
      
</testsuite>
